openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -28,29 +28,31 @@
 - control_key: AU-2
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-155
+        OpenShift has an API audit system which would need to be enabled
+        and configured for the organizational requirmennts.
     - key: b
       text: |
         Coordinating the security audit function with other
         organizational entities is outside the scope of
         system configuration.
     - key: c
       text: |
+        Coordinating the security audit function with other
+        organizational entities is outside the scope of
+        system configuration.
+
         For rationale regarding why the auditable events are deemed to be
         adequate to support after-the-fact investigations of security incidents,
         please contact your countries Common Criteria representative:
         https://www.commoncriteriaportal.org/ccra/members/
     - key: d
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-155
+        OpenShift has an API audit system which would need to be enabled
+        and configured for the organizational requirmennts.
 
```

Comment:
Response to this control is a gray area. We're often asked to document what events OpenShift will audit in a default installation, which would be part of a control response here (e.g. "every time a user does a deployment, pushes code, or deletes a pod, an audit event is triggered in the default installation of OpenShift. In addition to the default audit events, the NIST National Checklist for OpenShift 4.x adds the following.....")

Ref NIST language:
``d. Determines that the following events are to be audited within the information system: [Assignment: organization-defined audited events (the subset of the auditable events defined in AU-2 a.) along with the frequency of (or situation requiring) auditing for each identified event].``

Reply:
Documenting what default audit events are enabled would be a great way to make incremental progress on this control.